Source/Core/Core/PowerPC/SignatureDB/SignatureDB.h

```diff
@@ -18,6 +18,51 @@ class SignatureDBFormatHandler;
 class SignatureDB
 {
 public:
+  enum class Handler
+  {
+    DSY,
+    CSV
+  };
+  SignatureDB(Handler handler = Handler::DSY);
```

Comment:
I need a default constructor for all the original cases where using the default totaldb.dsy signature file. The format handler needs to know what kind of database it should use when constructed.